Source/WebCore/page/Page.cpp

```diff
@@ -5478,7 +5478,7 @@ std::optional<TextIndicatorData> Page::textPreviewDataForActiveWritingToolsSessi
         return std::nullopt;
     }
 
-    return IntelligenceTextEffectsSupport::textPreviewDataForRange(*localTopDocument, *scope, rangeRelativeToSessionRange);
+    return IntelligenceTextEffectsSupport::textPreviewForRange(*localTopDocument, *scope, rangeRelativeToSessionRange)->data();
```

Comment:
this should be renamed back to `textPreviewDataForRange` since I have been using `textPreview` itself to refer to the actual UIImage/NSImage